lib/ansible/module_utils/facts/hardware/freebsd.py

```diff
@@ -173,8 +173,11 @@ def get_device_facts(self):
 
         sysdir = '/dev'
         device_facts['devices'] = {}
-        drives = re.compile(r'(ada?\d+|da\d+|a?cd\d+)')  # TODO: rc, disks, err = self.module.run_command("/sbin/sysctl kern.disks")
-        slices = re.compile(r'(ada?\d+s\d+\w*|da\d+s\d+\w*)')
+        # TODO: rc, disks, err = self.module.run_command("/sbin/sysctl kern.disks")
+        drives = re.compile(r'(ada?\d+|da\d+|a?cd\d+|amrd\d+|da\d+|idad\d+|ipsd\d+|md\d+|mfid\d+|mlxd\d+|twed\d+|vtbd\d+)')
+        slices = re.compile(
+            (r'(ada?\d+[ps]\d+\w*|a?cd\d+[ps]\d+\w*|amrd\d+[ps]\d+\w*|da\d+[ps]\d+\w*|idad\d+[ps]\d+\w*|ipsd\d+[ps]\d+\w*'
+             r'|md\d+[ps]\d+\w*|mfid\d+[ps]\d+\w*|mlxd\d+[ps]\d+\w*|twed\d+[ps]\d+\w*|vtbd\d+[ps]\d+\w*)'))
```

Comment:
Readability tip: It is possible to use multiline raw-strings for and spread the regex across lines — https://docs.python.org/3/library/re.html#re.VERBOSE. As in
```python
r"""(?x)
\w+  # comment 1
\s+
\d+  # comment 2
"""
```

Reply:
@lostsoulparty Any updates?